scripts/db_connect.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+# This script requires google's cloud_sql_proxy on your PATH. One way to set this up:
+# $ brew cask install google-cloud-sdk
+# $ gcloud components install cloud_sql_proxy
+# $ PATH="$PATH:/usr/local/Caskroom/google-cloud-sdk/latest/google-cloud-sdk/bin"
+# OR
+# $ ln -s /usr/local/Caskroom/google-cloud-sdk/latest/google-cloud-sdk/bin/cloud_sql_proxy /usr/local/bin/cloud_sql_proxy
+#
+# Use this script to connect to cloud Postgres for a specific Catalog instance.
+# For example, to connect to the dev catalog database, run:
+# $ ENV=dev ./db-connect.sh
```

Comment:
If you have an existing `ENV`, what does this do?

Reply:
If you set an environment variable before a command, then that env var is only set for that command (vs exporting it, which will make it visible to all commands). If the env var is already set, the value set on the command line takes precedence.

Reply:
```sh
$ cat test.sh
echo $X
$ export X="hello"
$ ./test.sh
hello
$ X="goodbye" ./test.sh
goodbye
$
```

Reply:
Perfect. Although weirdly when I ran:
```
$ export foo="bar"   
$ foo="baz" echo $foo
bar
```
But I think that sys level commands might work differently

Reply:
When you write `foo="baz" echo $foo`, the `$foo` substitution occurs before the anything in the command line is executed. This can be useful sometimes, e.g. `PATH=$PATH:/tmp/bin something`. If `echo` itself supported a way to print an environment variable then you could change `$foo` on the command line:
```sh
$ export X="hello"
$ echo "" | X="goodbye" awk '{print ENVIRON["X"]}'
goodbye
```